<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QWidget" name="topBarWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="itemButton1">
        <property name="text">
         <string>主页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="itemButton2">
        <property name="text">
         <string>节目制作</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="itemButton3">
        <property name="text">
         <string>节目管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="itemButton4">
        <property name="text">
         <string>素材节目</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="itemButton5">
        <property name="text">
         <string>终端管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="itemButton6">
        <property name="text">
         <string>系统管理</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>362</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="userWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="logoWidget" native="true">
     <layout class="QGridLayout" name="gridLayout"/>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QStackedWidget" name="centralStackedWidget">
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
